src/SignaturePad.Shared/SignaturePadView.cs

```diff
@@ -49,6 +55,9 @@ partial class SignaturePadView
 #elif WINDOWS_UWP
 		private static readonly NativeColor SignaturePadDarkColor = Windows.UI.Colors.Black;
 		private static readonly NativeColor SignaturePadLightColor = Windows.UI.Colors.White;
+#elif NET471
+		private static readonly NativeColor SignaturePadDarkColor = new NativeColor () { A = System.Drawing.Color.Black.A, B = System.Drawing.Color.Black.B, G = System.Drawing.Color.Black.G, R = System.Drawing.Color.Black.R };
+		private static readonly NativeColor SignaturePadLightColor = new NativeColor () { A = System.Drawing.Color.White.A, B = System.Drawing.Color.White.B, G = System.Drawing.Color.White.G, R = System.Drawing.Color.White.R };
```

Comment:
For this, there is the https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.colors?view=netframework-4.7.1

Reply:
+